src/library/browse/browsefeature.cpp

```diff
@@ -21,6 +21,13 @@ namespace {
 
 const QString kQuickLinksSeparator = QStringLiteral("-+-");
 
+const QStringList linuxRemovableDrivePaths() {
+    QStringList paths;
+    paths.append("/media");
+    paths.append(QStringLiteral("/run/media/") + QString::fromLocal8Bit(qgetenv("USER")));
```

Comment:
It should be the same for `/media ` on Ubuntu/Pop-Os - `/media/$USER`

Reply:
I know, it's like that on Ubuntu (which Pop!OS is based on).
Though TLDP https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/media.html says (can't tell how up-to-date that is, and which distros have their own rules, but /media seems to be the common base):
> The following directories, or symbolic links to directories, must be in /media, if the corresponding subsystem is installed:
>
>floppy     Floppy drive (optional)
cdrom      CD-ROM drive (optional)
cdrecorder CD writer (optional)
zip        Zip drive (optional)
>
>On systems where more than one device exists for mounting a certain type of
media, mount directories can be created by appending a digit to the name of
those available above starting with '0', but the unqualified name must also
exist.
>
> A compliant implementation with two CDROM drives might have /media/cdrom0
and /media/cdrom1 with /media/cdrom a symlink to either of these.